<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_box</class>
 <widget class="QDialog" name="chat_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>581</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_1_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1_1">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>5</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>13</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>580</width>
      <height>460</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_2_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>150</width>
       <height>460</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>430</width>
       <height>275</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>2021/2/9 12:57:18</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>430</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_10">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3_9">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>335</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2_1">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭(C)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>发送(S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_selections">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>420</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
